packages/adblocker/src/engine/engine.ts

```diff
@@ -354,6 +378,14 @@ export default class FilterEngine extends EventEmitter<
     }
   }
 
+  public isFilterExcluded(filter: IFilter): boolean {
+    return this.preprocessors.isFilterExcluded(filter);
+  }
```

Comment:
I don't think this needs to be part of the public API. It could either be private or we remove the method completely and use `this.preprocessors.isFilterExcluded.bind(this.preprocessors)` at call site.

Reply:
Resolved by [`0f7495d` (#3782)](https://github.com/ghostery/adblocker/pull/3782/commits/0f7495d4908591263dded2d5eef661854c0817a2)